cardinal/config.go

```diff
@@ -160,3 +189,52 @@ func (w *WorldConfig) setLogger() error {
 
 	return nil
 }
+
+func setupViper() {
+	once.Do(func() {
+		// Bind the command-line flags to Viper
+		pflag.String(tomlConfigFileEnvVariable, "", "path to the toml config file")
```

Comment:
Make sure to properly capitalize the message here since it's a user facing message

Reply:
updated